packages/central-server/src/index.js

```diff
@@ -37,7 +37,7 @@ const models = new ModelRegistry(database, modelClasses, true);
 /**
  * Set up change handlers e.g. for syncing
  */
-if (isFeatureEnabled('MEDITRAK_SYNC_QUEUE')) {
+if (isFeatureEnabled('MEDITRAK_SYNC_QUEUE', FEATURE_FLAG_ON)) {
```

Comment:
I'm not totally sure this is the right place to have the feature flag defaulting. With more involved feature flagging, we might have conditional checks in multiple places, and it'd be better to have one place that determined whether that feature flag was defaulted to on or off. That might look like a central repository somewhere else that defines the defaults - which in an ideal world would look like moving to https://www.npmjs.com/package/config instead of .env files, like Tamanu does :-)

Reply:
Yeah good point! I thought about maybe adding a fairly lightweight `@tupaia/feature-toggles` package, which I do still feel might be the way to go...

Alternatively, how about an even simpler option in `@tupaia/utils/src/featureToggles.js`:
```
const toggles = {
  MEDITRAK_SYNC_QUEUE: {
    default: FEATURE_FLAG_ON
  },
  SERVER_CHANGE_ENQUEUER: {
    default: FEATURE_FLAG_OFF
  },
}

const isFeatureEnabled = (featureName) => {
  if (!(featureName in toggles)) {
    throw new Error (`Unkown feature: ${featureName});
  }
  
  let toggleState;
  if (toggles[featureName].default) {
    toggleState = getEnvOrDefault(featureName, toggles[featureName].default);
  } else {
    toggleState = requireEnv(featureName);
  }

  return toggleState.toLowerCase() === FEATURE_FLAG_ON;
}
```

Reply:
Yup I like the simpler option, at least while there are only two and the behaviour is really limited

Reply:
A note to get it off my mind: I was just wondering whether we should keep the toggle definitions in the package they apply to, but it feels like over-engineering to have three separate files just to manage this small amount of toggling. It could be something to keep in mind if this grows though, but there could well be toggles that apply across packages too.

Reply:
@edmofro I thought about that too with keeping the toggles in their relevant packages, but I couldn't shake the feeling that it could be fairly common for a toggle to be spread across packages (this one sort of is in a way) and so didn't want to lock it in to that kind of pattern.